mobile/native/src/api.rs

```diff
@@ -213,6 +214,26 @@ pub fn force_close_channel() -> Result<()> {
     ln_dlc::close_channel(true)
 }
 
+/// Returns channel info if we have a channel available already
+///
+/// If no channel is established with the coordinator `None` is returned.
+pub fn channel_info() -> Result<Option<ChannelInfo>> {
```

Comment:
consider just returning just `Option<ChannelInfo>` - do you really need to distinguish why there's no channel info?

Reply:
I think we should distinguish between a channel not existing yet `Ok(None)` and errors; those are completely different scenarios and a users might have to be made aware of errors (eventually).